homeassistant/components/switch/rpi_rf.py

```diff
@@ -0,0 +1,94 @@
+"""
+Allows to configure a switch using a 433MHz module via GPIO on a Raspberry Pi.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.rpi_rf/
+"""
+
+import logging
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.const import CONF_VALUE_TEMPLATE
+
+REQUIREMENTS = ['RPi.GPIO==0.6.2', 'rpi-rf==0.9.5']
```

Comment:
No need to add `RPi.GPIO` here as it will be installed as a dependency of `rpi-rf`